sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsFailedDetailsParser.cs

```diff
@@ -20,84 +20,73 @@ public override bool TryParse(Response response, out ResponseError? error, out I
             data = default;
             error = default;
 
-            if (response.ContentStream is { CanSeek: true })
```

Comment:
Takes advantage of new buffering logic introduced in https://github.com/Azure/azure-sdk-for-net/pull/42294